.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -126,6 +124,7 @@ RUN git checkout "${OWLBOT_CLI_COMMITTISH}"
 RUN npm i && npm run compile && npm link
 RUN owl-bot copy-code --version
 RUN chmod o+rx $(which owl-bot)
+RUN apk del -r npm && apk cache clean
```

Comment:
I like this approach! We don't need `npm` in our image as long as `owl-bot` package is already built.